azure-pipelines/azure-pipelines.yml

```diff
@@ -249,10 +249,14 @@ stages:
           -t edgehub/deviceshifu-http-opcua:$(tag) --load
         displayName: build deviceshifu-http-opcua
       - script: |
-          docker buildx build --platform=linux/amd64 \
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+          docker buildx create --use
+        displayName: configure multi-arch and buildx
+      - script: |
+          docker buildx build --platform=linux/amd64,linux/arm64,linux/arm \
           -f $(System.DefaultWorkingDirectory)/examples/deviceshifu/mockdevice/opcua/Dockerfile.mockdevice-opcua \
           $(System.DefaultWorkingDirectory) \
-          -t edgehub/opcua-mock-device:$(tag) --load 
+          -t edgehub/opcua-mock-device:$(tag)
```

Comment:
what's the difference between --load and without it.

Reply:
nvm, was testing earlier